constraint/pkg/client/client.go

```diff
@@ -44,19 +44,12 @@ type Client struct {
 	AllowedDataFields []string
 }
 
-// createDataPath compiles the data destination: data.external.<target>.<path>.
-func createDataPath(target, subpath string) string {
-	subpaths := strings.Split(subpath, "/")
-	p := []string{"external", target}
-	p = append(p, subpaths...)
-
-	return "/" + path.Join(p...)
-}
-
-// AddData inserts the provided data into OPA for every target that can handle the data.
+// AddCachedData inserts the provided data into OPA for every target that can handle the data.
```

Comment:
Instead of "into OPA ..." - should be "every target and driver which caches data". For example, the gatekeeper cache doesn't (or rather, won't) use OPA and neither does the test target. Ditto for RemoveCachedData.